src/mame/audio/seibu.h

```diff
@@ -44,11 +44,15 @@ class seibu_sound_device : public device_t
 	seibu_sound_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
 	~seibu_sound_device() {}
 
-	// static configuration
-	static void static_set_cpu_tag(device_t &device, const char *tag);
-	static void static_set_rombank_tag(device_t &device, const char *tag);
-	template<class _Object> static devcb_base &set_ym_read_callback(device_t &device, _Object object)  { return downcast<seibu_sound_device &>(device).m_ym_read_cb.set_callback(object); }
-	template<class _Object> static devcb_base &set_ym_write_callback(device_t &device, _Object object) { return downcast<seibu_sound_device &>(device).m_ym_write_cb.set_callback(object); }
+	//  configuration
+	void set_cpu_tag(const char *tag)
+	{
+		m_sound_cpu.set_tag(tag);
+		m_sound_rom.set_tag(tag);
+	}
+	void set_rombank_tag(const char *tag) { m_rom_bank.set_tag(tag); }
+	template<class Object> devcb_base &set_ym_read_callback(Object&& object)  { return m_ym_read_cb.set_callback(std::forward<Object>(object)); }
```

Comment:
When you're doing this, could you please be consistent with the style used in the rest of MAME?  We put right-associative things to the right of the space.

Note that in this line it's `devcb_base &set_ym_read_callback` (the ampersand qualifying it as an lvalue reference is after the space).  Likewise, it should be `Object &&object` (with the double ampersand qualifying it as a universal reference coming after the space).

Also, this stuff is less confusing if there aren't different things called `Object` and `object` in the same line - that's why I've been changing them to `cb` or `callback` when I change them.